contracts/test/v0.1/RequestResponseConsumerContract.test.cjs

```diff
@@ -159,7 +188,46 @@ async function requestAndFulfill(
       })
     ).wait()
   } else {
-    const { accId } = await createAccount(prepayment.contract, consumer.signer)
+    let accId
+    const startTime = Math.round(new Date().getTime() / 1000)
+    const period = 1000 * 60 * 60 * 24 * 7
```

Comment:
I am curious what does the 1000 mean here?

`60 * 60 * 24 * 7` should represent 7 days in seconds, but with 1,000, it would be 1,000 weeks, am I wrong?

Reply:
Yes, I was about to set 1 week, got lost between seconds in JS and millisecond in solidity